server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -20,23 +20,37 @@ alfred:
   sessionStickinessDurationMs: 3600000
   enableConnectionCountLogging: false
   throttling:
-    restCallsPerTenant:
-        maxPerMs: 1000000
-        maxBurst: 1000000
-        minCooldownIntervalInMs: 1000000
-        minThrottleIntervalInMs: 1000000
-        maxInMemoryCacheSize: 1000
-        maxInMemoryCacheAgeInMs: 60000
-        enableEnhancedTelemetry: false
-    restCallsPerCluster:
-        createDoc: 
+    tenantGroup1:
```

Comment:
This naming is a little confusing. It would be good to mention somewhere that tenantGroup limits are still per Tenant and not per group.

Reply:
Another thought: since we can only have 1 tenantGroup right now (because this isn't extensible right now), should we call this exceptionTenants, or something similar?

Reason -> i feel saying tenantGroup1 without having flexibility of creating another tenantGroup is misleading.

Reply:
Change the name to tenantsGroup1PerTenant.
Also, there is a possibility to scale out tenants group, such as tenantsGroup2PerTenant, and so on.

Reply:
Correct but to do that, we will also have to update runnerFactory as well. So, it won't just be a config update, we will also have to change the code.

so essentially right now, with this pr code change, we can only support 1 tenant Group

Reply:
Yes. We can keep it as it is right now. :) If we want to add more tenant groups, we will change the code.

Reply:
Reason: If we call it exceptionTenants right now, and add more tenant groups later, we would also change the code as well. :)